include/kernels/blas_3d.cuh

```diff
@@ -0,0 +1,341 @@
+#pragma once
+
+#include <color_spinor_field_order.h>
+#include <index_helper.cuh>
+#include <fast_intdiv.h>
+#include <quda_matrix.h>
+#include <matrix_field.h>
+#include <kernel.h>
+//#include <kernels/contraction_helper.cuh>
```

Comment:
This can be deleted

Reply:
3767371511982bbe8a6ec4cb23a09dda501b6272